pwa/src/services/isUUID.ts

```diff
@@ -0,0 +1,9 @@
+export const isUUID = (id: string): boolean => {
+  const regexExp = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/gi;
+
+  if (regexExp.test(id)) {
+    return true;
+  } else {
+    return false;
+  }

```

Comment:
Would `return regexExp.test(id)` effectively do the same as the `if / else` statement?